assets/css/edd-admin.css

```diff
@@ -1251,10 +1251,10 @@ textarea[name="edd-note"] {
 	margin-top: -30px;
 	margin-left: 2px;
 }
-
+/* Removed by Robin -- first selector only*/
 #edd-filters input[type="text"].edd_datepicker,
 #edd-filters input[type="number"] {
-	max-width: 100px;
+	max-width: 105px;
```

Comment:
This won't take effect until the `width: 100px;` rule is removed (lines 2465-2467), as that's overriding this.

Reply:
My screen is showing the fields as `105px`.  Where are you seeing it being overridden in the UI?

Reply:
It looks like `max-width` overrides `width` https://developer.mozilla.org/en-US/docs/Web/CSS/max-width

Reply:
Sorry, the filter bar inputs are showing at 100px--the ones in the export form do not appear to be affected.

Reply:
Also, totally not sure why the filter bar is showing on the Exports screen--does that seem weird? The filters don't affect anything there as far as I can tell.